HW0/P4/P4.py

```diff
@@ -51,12 +58,31 @@
     # A = ?
     # a = ?
     # s = ?
-
+    
+    A = np.array( [ [1,0,0, dt,  0,  0], 
+                    [0,1,0,  0, dt,  0],
+                    [0,0,1,  0,  0, dt],
+                    [0,0,0, 1-c*dt, 0, 0],
+                    [0,0,0, 0, 1-c*dt, 0],
+                    [0,0,0, 0, 0, 1-c*dt] ])
+    
+    a = np.array( [0,0,0,0,0,g*dt] ).reshape([-1,1])
+    
+    s_prop = np.empty([6,K])
+    
     # Initial conditions for s0
+    s0 = np.array([0,0,2,15,3.5,4.0]).reshape([-1,1])
```

Comment:
Instead of hard-coding the parameters, you could reference s_true[0,:] instead.  That way you only have to update the values in one place if they change.